ui/pause_menu.gd

```diff
@@ -0,0 +1,37 @@
+extends Node3D
+
+@onready var animation_player: AnimationPlayer = %AnimationPlayer
```

Comment:
Is this a global animation player? If so, can we give it a name more fitting? otherwise, if its local to this scene, probably use `$AnimationPlayer` and remove the global nametag